fotoobo/inventory/inventory.py

```diff
@@ -49,16 +51,18 @@ def get(
         Get a list of assets from the inventory
 
         Args:
-            name (str, optional): Which asset to get. If blank get every asset
-            type (str, optional) : Which asset type to get. If blank get every asset.
+            name:   Which asset to get. If blank get every asset. Wildcard * is supported in any
+                    position. E.g. "FortiGate", "*Gate", "Forti*ate", "Forti*".
+            type:   Which asset type to get. If blank get every asset.
 
         Returns:
-            Dict[str, Any]: Dict of assets with name as key
+            Dict of assets with name as key
+
+        Raises:
+            GeneralWarning if no asset was found that matches name or type
         """
         log.debug("getting assets with name '%s' and type '%s'", name, type)
-        if name and name not in self.assets:
-            raise GeneralWarning(f"asset {name} is not defined in inventory")
-
+        name_pattern = f"^{name}$".replace("*", ".*")
```

Comment:
`>>> name = None`
`>>> name_pattern = f"^{name}$".replace("*", ".*")`
`>>> name_pattern`
`'^None$'`
I think it's not really a problem, because name_pattern is not used if name id None

Reply:
ok to me.